.changeset/pre.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Aha! This will be the real last comment. Should the `pre` take be removed or are you intending another prerelease?

Reply:
Yes, will remove.